configuration/manage-connection.md

```diff
@@ -0,0 +1,38 @@
+---
+description: This article shows how to modify and manage your tenant connection in Syskit Point.
+---
+
+# Manage Tenant Connection
+
+{% hint style="info" %}
+* Point Cloud customers can manage and edit their tenant connection after deploying Syskit Point.
+
+* Point Data Center customers who set up their tenant connection when deploying Syskit Point can manage or edit their connection by following the steps in this article. 
+{% endhint %}
+
+Once Syskit Point is set up, you can manage or modify your tenant connection anytime through your Syskit Point Settings. This allows you to adjust permissions as needed and ensure accurate data collection. 
```

Comment:
adjust permissions bih zamijenio sa adjust autodiscover and audit data storing options ili nešto slično